creativecloud/blocks/catalog/catalog.js

```diff
@@ -65,6 +65,15 @@ export function enableAnalytics(catalog, merchCards, sidenav) {
   });
 }
 
+function removeAriaHidden(sidenav) {
```

Comment:
I think this should be addressed in the sidenav block in CC:
https://github.com/adobecom/cc/blob/stage/creativecloud/blocks/sidenav/sidenav.js#L155

Reply:
Yes, you are right. Of course. :) And first I tried to do it in sidenav.js where that element is created and it didn't work so I did it in catalog.js. But now I also know why. Done.

Reply:
Yes @yesil, you are right. Of course. :) And first I tried to do it in sidenav.js where that element is created and it didn't work so I did it in catalog.js. But now I also know why. Done.